### DISCUSSION TITLE
Listing idendical tours

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1131] [General]
Hi
Is there any option that would allow me to select and list several identical tours? That would enable me to see which one on a given period of time, is the best average speed  (for example) at a glance!
Thank you

**Comment by wolfgang-ch** [comment]
https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tool-compare-tours
https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tool-compare-tours-by-geo

The next version will have some improvement for the geo compare tool
